Rest.Services/Repositories/ProductRepository.cs

```diff
@@ -0,0 +1,37 @@
+using Rest.Models;
+namespace Rest.Services;
+public class ProductRepository : IRepository<long, Product>
+{
+    public Dictionary<long, Product> _productDictionary;
+    public long _id;
```

Comment:
Same here. In addition, 
* what is this for??
* Is a field?

Reply:
This ```_id``` is a private field that will increment each time I add a new object to the repository. So, it's responsible of the dictionary's ID.